core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -1043,6 +1049,8 @@ class TimeSeriesShard(val ref: DatasetRef,
         // add new lucene entry if this partKey was never seen before
         // causes endTime to be set to Long.MaxValue
         partKeyIndex.addPartKey(newPart.partKeyBytes, partId, startTime)()
+        // Track new partkeys which are added to index.
+        shardStats.partKeyIndexAdded.increment()
```

Comment:
this will be emmitted during both kafka recovery and normal ingestion.

Reply:
Only during normal ingestion.